api/controllers/register.js

```diff
@@ -68,6 +68,7 @@ class Register {
         transaction: transaction
       })
 
+      name = name.trim()
```

Comment:
This should probably happen *after* we replace `CMDR`, since we may have an extra space to remove in there afterwards.